components/script/dom/servoparser/mod.rs

```diff
@@ -737,6 +742,23 @@ impl FetchResponseListener for ParserContext {
             parser.parse_sync();
         }
     }
+
+    fn resource_timing(&mut self) -> &mut ResourceFetchTiming {
+        &mut self.resource_timing
+    }
+
+    fn submit_resource_timing(&self) {
+        match self.parser {
+            Some(ref parser) => {
+                let document = &*parser.root().document;
+                let local_name = DOMString::from("other");
+                let performance_entry = PerformanceResourceTiming::new(
+                    &document.global(), self.url.clone(), local_name, None, &self.resource_timing);
+                document.global().performance().queue_entry(performance_entry.upcast::<PerformanceEntry>(), false);
```

Comment:
ParserContext is the listener for a document navigation, so I don't think we want to create a new entry for that.